Build/images/samples/Dockerfile

```
@@ -2,7 +2,7 @@
 # https://github.com/microsoft/iqsharp/blob/master/images/iqsharp-base/Dockerfile.
 # As per Binder documentation, we choose to use an SHA sum here instead of a
 # tag.
-FROM mcr.microsoft.com/quantum/iqsharp-base:0.11.2004.2825
+FROM mcr.microsoft.com/quantum/iqsharp-base:0.12.20070124
```

Not too important, but the latest version is actually `0.12.20072031`.
```suggestion
FROM mcr.microsoft.com/quantum/iqsharp-base:0.12.20072031
```

---

Thanks for catching that!